src/constants/skip.ts

```diff
@@ -0,0 +1,33 @@
+export type RouteStatus = {
+  chainId: string | undefined;
+  txHash: string | undefined;
+  status: string | undefined;
+};
+export type SkipTransactionStatus = {
+  chainData: {
+    chainId: string | undefined;
+    chainName: string | undefined;
+    estimatedRouteDuration?: string | undefined;
```

Comment:
Would it be easier to have `estimatedRouteDuration` this as number, so we don't have to special case for it later?

Reply:
yeah i was thinking about this too. @tyleroooo also brought this up. 
the issue is that we don't have an estimatedRouteDuration anymore, so the thought was that we'd say the transfer is coming in "< 30 minutes". I can think a little bit longer on how we can do this without messing up the typing too badly

Reply:
FWIW (to @tyleroooo too) this is temporary until skip implements the `estimatedRouteDuration` property into their api response